src/client/statics/Error401.js

```diff
@@ -18,16 +18,16 @@ const Error401 = () => (
         </h2>
         <p>
           <FormattedMessage
-            id="@statics/need_login"
+            id="need_login"
             defaultMessage="You need to login to use this feature, please"
           />{' '}
           <a href={SteemConnect.getLoginURL()}>
             <FormattedMessage id="login" defaultMessage="Login" />
           </a>
           <br />
-          <FormattedMessage id="@statics/here_link" defaultMessage="Here's a link to the" />
+          <FormattedMessage id="here_link" defaultMessage="Here's a link to the" />
```

Comment:
I think there might be a case where this order wouldn't make sense (in some languages).
I think we can embed link inside `here_link`.